### PR TITLE
Updated integration tests for Copy Group

### PR DESCRIPTION
**Pull request by smmr-dn**
Added Grouping and Mapping base URL for mappingClients in integration test

### CODE REVIEW

File: src/integration-test/utils/imodels-client-test-utils/BaseIntegrationTestsConfig.ts [hunk 1]
```diff
@@ -11,6 +11,7 @@ export interface AuthConfigValues {
 
 export interface ApiConfigValues {
   baseUrl: string;
+  groupingAndMappingBaseUrl?: string;
```
Comment (MindaugasButkus) [inline src/integration-test/utils/imodels-client-test-utils/BaseIntegrationTestsConfig.ts:14]: Revert this change.

File: src/integration-test/utils/IModelsClientsTestsConfig.ts [hunk 2]
```diff
@@ -43,6 +43,7 @@ export class IModelsClientsTestsConfig implements BaseIntegrationTestsConfig {
       },
       reporting: {
         baseUrl: process.env.APIS_REPORTING_BASE_URL ?? REPORTING_BASE_PATH,
+        groupingAndMappingBaseUrl: process.env.APIS_GROUPING_AND_MAPPING_BASE_URL ?? GROUPING_AND_MAPPING_BASE_PATH,
```
Comment (MindaugasButkus) [inline src/integration-test/utils/IModelsClientsTestsConfig.ts:46]: Add a new property to the parent object - `groupingAndMapping`.

**Comment by MindaugasButkus** [comment]
Update README.md.

File: src/integration-test/utils/imodels-client-test-utils/BaseIntegrationTestsConfig.ts (hunk 1 above)
Comment (smmr-dn) [reply src/integration-test/utils/imodels-client-test-utils/BaseIntegrationTestsConfig.ts]: I have reverted this change

File: src/integration-test/utils/IModelsClientsTestsConfig.ts (hunk 2 above)
Comment (smmr-dn) [reply src/integration-test/utils/IModelsClientsTestsConfig.ts]: The new property has been added

**Review by MindaugasButkus — APPROVED** [review]
